store/index.js

```diff
@@ -390,16 +407,22 @@ export const getters = {
 
                 // 1. Determine if at least one column has been linked to a category
                 const categorizationStatus = Object.values(p_state.columnToCategoryMapping)
-                                                   .some(category =>  null !== category );
+                                                   .some(category => null !== category );
 
                 // 2. Make sure one (and only one) column has been categorized as 'Subject ID'
                 const singleSubjectIDColumn = ( 1 === Object.values(p_state.columnToCategoryMapping)
                                                             .filter(category => "Subject ID" === category)
                                                             .length );
 
+                const notOnlySubjectIDCategorized = ( Object.values(p_state.columnToCategoryMapping)
+                                                            .filter(category => "Subject ID" !== category &&
+                                                                    null !== category)
+                                                            .length >= 1 );
+
                 // Annotation page is only accessible if at least one column has
                 // been categorized and if one (and only one) column has been categorized as 'Subject ID'
-                pageAccessible = categorizationStatus && singleSubjectIDColumn;
+                // and if at least one category other than Subject ID has been categorized
+                pageAccessible = categorizationStatus && singleSubjectIDColumn && notOnlySubjectIDCategorized;
```

Comment:
Is there a case where `singleSubjectIDColumn` and `notOnlySubjectIDCategorized` are `true` but `categorizationStaus` is `false`?

Reply:
Good catch. With the new page access criteria, `categorizationStatus` can now be removed.